submissions/vincentNjele/Dom/vendors.js

```diff
@@ -0,0 +1,223 @@
+
+
+var addToCart = document.getElementById('addtocart');
+var addToCart2 = document.getElementById('addtocartss2');
+var mycounter= 1;
+
+
+var notification = document.getElementById('noti');
+var view = window.innerWidth;
+
+var counter = 1;
+
+if(view<=768 ) {
+        addToCart.style.cursor = "pointer";
+        addToCart2.style.cursor= "pointer";
+       
+}
+
+
+addToCart.onclick = HandleFunction;
+
+function HandleFunction() {
+    
+    if(counter>0) {
+        
+        if(view<768){
+
+            addToCart.style.width = "150px";
```

Comment:
It is not good coding practice that your styles should be inside your html or js files